pkg/cmd/run.go

```diff
@@ -262,4 +271,16 @@ func bindFlags(cmd *cobra.Command) {
 
 	cmd.Flags().String("log-format", defaultConfig.Log.Format, "the log format to output logs in")
 	cmdutil.MustBindPFlag("log.format", cmd.Flags().Lookup("log-format"))
+
+	cmd.Flags().Int("max-tuples-per-write", defaultConfig.OpenFGA.MaxTuplesPerWrite, "the maximum allowed number of tuples per Write transaction")
+	cmdutil.MustBindPFlag("maxTuplesPerWrite", cmd.Flags().Lookup("max-tuples-per-write"))
+
+	cmd.Flags().Int("max-types-per-authz-model", defaultConfig.OpenFGA.MaxTypesPerAuthorizationModel, "the maximum allowed number of type definitions per authorization model")
```

Comment:
Can we use authorization-model instead of authz-model?  Prefer to have 1-1 mapping with the .config-schema.json?  If we want to have authz-model, then update config-schema.json accordingly.

Reply:
In my mind, I always just call it a `model` now because this is an authorization app. My vote would be (if we are voting 😉) to rename them all to `model` 🤪 .